ambry-router/src/main/java/com.github.ambry.router/GetBlobOperation.java

```diff
@@ -85,9 +87,11 @@
   // the total number of data chunks retrieved so far (and may or may not have been written out yet).
   private int numChunksRetrieved;
   // the maximum size of a data chunk in bytes
-  private int chunkSize = CompositeBlobInfo.UNDEFINED_CHUNK_SIZE;
+  private long chunkSize;
```

Comment:
minor: why change this from `int` to `long`? The config `routerMaxPutChunkSizeBytes` (from where this is derived) is an `int`.